src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.Node.cs

```diff
@@ -72,5 +78,107 @@ private string GetDebuggerDisplay()
                 return NameSpan + ", " + ParentIndex;
             }
         }
+
+        private readonly struct ParameterTypeInfo
+        {
+            /// <summary>
+            /// This is the type name of the parameter when <see cref="IsComplexType"/> is true.
```

Comment:
wait... i thought this was used when it *wasn't* a complex type.

Reply:
Oops. Fixed